cal/cal/static/js/app.js

```diff
@@ -6,21 +6,22 @@ var analyticsApp = window.angular.module('analyticsApp', ['analyticsApp.services
 analyticsApp.controller('LoggedInCtrl', function LoggedInController() {
 });
 
-function TagListCtrl($scope, $http, CalendarRangeService, TagService) {
+function TagListCtrl($scope, $http, CalendarFilterService, TagService) {
 
   var _this = this;
 
   this.tags = [];
   this.tags.dataLoaded = false;
 
-  $scope.$on('calendarRange:updated', function(event, data) {
+  $scope.$on('calendarFilter:updated', function(event, data) {
```

Comment:
can you do a bind(this) instead of using _this? it'll align more with how it's been done in the past